gradle.properties

```diff
@@ -38,6 +38,7 @@ hamcrestCoreVersion = 1.3
 jacksonVersion = 2.10.2
 jcommanderVersion = 1.78
 jgitVersion = 5.6.0.201912101111-r
+jSchAgentProxyVersion = 0.0.7
```

Comment:
This should be version 0.0.9, and I'd still prefer `jschAgentProxyVersion` over `jSchAgentProxyVersion` (as we do for JGit).